docs/admin-guide/provisioning/kind.md

```diff
@@ -4,21 +4,22 @@
 
 Download the latest official binary release for your platform from the [github repository](https://github.com/flanksource/karina/releases/latest).
 
-=== "Linux"
+#### Linux
 ```bash
-wget https://github.com/flanksource/karina/releases/download/v0.15.1/karina
+wget https://github.com/flanksource/karina/releases/download/v0.24.4/karina
 chmod +x karina
```

Comment:
```suggestion
wget -nv -nc -O karina https://github.com/flanksource/karina/releases/latest/download/karina && chmod +x karina
```

!!! info
    For production pipelines you should always pin the version of karina you are using

Reply:
Nice!